addon/mongocrypt.h

```diff
@@ -67,6 +69,9 @@ class MongoCrypt : public Napi::ObjectWrap<MongoCrypt> {
     Napi::Value MakeExplicitDecryptionContext(const Napi::CallbackInfo& info);
     Napi::Value MakeDataKeyContext(const Napi::CallbackInfo& info);
     Napi::Value MakeRewrapManyDataKeyContext(const Napi::CallbackInfo& info);
+    Napi::Value MakeExplicitEncryptionContextInternal(ExplicitEncryptionContextInitFunction init_fn,
+                                                      const Napi::Uint8Array& value,
+                                                      const Napi::Object& options);
```

Comment:
Also for my own knowledge, do we expose this in the header file just for unit-testing purposes?

Reply:
This is a private class member - as far as I know, you must declare a class member in the header before defining it.  (it needs to be a class member, at least as currently written, because it uses class data.  an alternate approach would be to have an internal function only in the .cpp file that takes all the data it needs as parameters).